.github/workflows/ci.yml

```diff
@@ -0,0 +1,26 @@
+on: [push, pull_request]
+name: Test
+jobs:
+  test:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, windows-latest]
+    runs-on: ${{matrix.os}}
+    steps:
+    - uses: actions/setup-go@v2
+      with:
+        go-version: 1.x
+    - run: |
+        go get golang.org/x/tools/cmd/goimports
+        go install github.com/mattn/goveralls@latest
+    - run: |
+        git config --global core.symlinks true
+    - uses: actions/checkout@v2
```

Comment:
Travis [clones submodules by default](https://docs.travis-ci.com/user/customizing-the-build/#git-submodules). Fixed in #196.